daemon/monitor.go

```diff
@@ -105,6 +105,10 @@ func (daemon *Daemon) handleContainerExit(c *container.Container, e *libcontaine
 		"exitCode":     strconv.Itoa(ctrExitStatus.ExitCode),
 		"execDuration": strconv.Itoa(int(execDuration.Seconds())),
 	}
+	if c.HasBeenManuallyRestarted {
```

Comment:
`c` is already locked as we enter this func, so we can safely read this attribute